app/native/src/components/authentication/EmailLoginForm.js

```diff
@@ -24,6 +24,29 @@ export default class SimpleLoginForm extends React.Component<Props, State> {
     loading: false,
   };
 
+  _handleEmailChange = text =>
+    this.setState({
+      username: text,
+    });
+
+  _handlePasswordChange = text =>
+    this.setState({
+      password: text,
+    });
+
+  _handleFormSubmit = () =>
```

Comment:
Out of scope now. Also, don't overuse Redux. It's not needed in this case because I am only changing state inside of this component. We can improve it later...